### ISSUE TITLE
crash after getting 5 drops, crash on no streamer online

### ISSUE THREAD

**Issue by skupfer** [Zaarrg/TTVDropBot#8]
Version 1.3.2.1

Crash during the night

```
(node:1856) UnhandledPromiseRejectionWarning: Error: Protocol error (Page.reload): Session closed. Most likely the page has been closed.
    at CDPSession.send (C:\Users\virtual\Desktop\TTVDropBot\node_modules\puppeteer\lib\cjs\puppeteer\common\Connection.js:195:35)
    at next (C:\Users\virtual\Desktop\TTVDropBot\node_modules\puppeteer-extra-plugin-stealth\evasions\sourceurl\index.js:30:43)
    at CDPSession.send (C:\Users\virtual\Desktop\TTVDropBot\node_modules\puppeteer-extra-plugin-stealth\evasions\sourceurl\index.js:33:18)
    at Page.reload (C:\Users\virtual\Desktop\TTVDropBot\node_modules\puppeteer\lib\cjs\puppeteer\common\Page.js:794:26)
    at MapDrops (C:\Users\virtual\Desktop\TTVDropBot\src\functions\MapDrops.js:5:21)
    at CheckProgressCurrentPage (C:\Users\virtual\Desktop\TTVDropBot\src\Checks\CheckProgressCurrentPage.js:13:11)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async C:\Users\virtual\Desktop\TTVDropBot\src\Pages\StreamPage.js:64:9
    at async CurrentProgressEvent (C:\Users\virtual\Desktop\TTVDropBot\src\Pages\StreamPage.js:62:5)
(Use `TTVDropBot --trace-warnings ...` to show where the warning was created)
(node:1856) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:1856) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Starting it again it doesn't find any drops (Checking for Live Channels is empty because all are offline except for iiTzTimmy where I got the drop already). If pressing enter at "Select Twitch Channel to start Watching?" it crashes to another error (I expected it will wait for someone to come online)

```
(node:11172) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'value' of undefined
    at ListPrompt.getCurrentValue (C:\Users\virtual\Desktop\TTVDropBot\node_modules\inquirer\lib\prompts\list.js:148:53)
    at MapSubscriber._next (C:\Users\virtual\Desktop\TTVDropBot\node_modules\rxjs\internal\operators\map.js:49:35)
    at MapSubscriber.Subscriber.next (C:\Users\virtual\Desktop\TTVDropBot\node_modules\rxjs\internal\Subscriber.js:66:18)
    at TakeSubscriber._next (C:\Users\virtual\Desktop\TTVDropBot\node_modules\rxjs\internal\operators\take.js:54:30)
    at TakeSubscriber.Subscriber.next (C:\Users\virtual\Desktop\TTVDropBot\node_modules\rxjs\internal\Subscriber.js:66:18)
    at Interface.handler (C:\Users\virtual\Desktop\TTVDropBot\node_modules\rxjs\internal\observable\fromEvent.js:22:28)
    at Interface.emit (events.js:327:22)
    at Interface._onLine (readline.js:337:10)
    at Interface._line (readline.js:666:8)
    at Interface._ttyWrite (readline.js:1010:14)
(Use `TTVDropBot --trace-warnings ...` to show where the warning was created)
(node:11172) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11172) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Aerosyth** [comment]
I have the same exact issue here. Running as admin doesn't seem to change the outcome either.

**Comment by Zaarrg** [comment]
Working on it, will take a while, probably will have to upload a new update to download. And the first error you got is because the bot couldnt reload the page, maybe because of your internet or chrome crashed. Cant really say any specifc about it.

**Comment by Zaarrg** [comment]
Error Fixed with release of [1.3.2.2 | Hotfix 2](https://github.com/Zaarrg/TTVDropBot/releases/tag/1.3.2.2). Bot should be now working again.
